src/transcode.c

```diff
@@ -281,6 +287,7 @@ stream_settings_set(struct stream_ctx *s, struct settings_ctx *settings, enum AV
       s->codec->channels       = settings->channels;
       s->codec->sample_fmt     = settings->sample_format;
       s->codec->time_base      = (AVRational){1, settings->sample_rate};
+      s->codec->bit_rate       = settings->bit_rate;
```

Comment:
this `AVCodecContext` value was never previously set and was populated by the default value (128000) - setting this to 0 has no impact on day to day transcode ops in testing